squirrel.api/pom.xml

```diff
@@ -99,31 +114,33 @@
             <groupId>org.aksw.jena-sparql-api</groupId>
             <artifactId>jena-sparql-api-core</artifactId>
         </dependency>
-        
-		        <!-- https://mvnrepository.com/artifact/org.rdfhdt/hdt-java-package -->
-		
-		<dependency>
-		    <groupId>org.rdfhdt</groupId>
-		    <artifactId>hdt-java-package</artifactId>
-		    <type>pom</type>
-		    <exclusions>
-		        <exclusion>
-		          <groupId>junit</groupId> <!-- Exclude Project-D from Project-B -->
-		          <artifactId>junit-dep</artifactId>
-		        </exclusion>
-		      </exclusions>
-		</dependency>
-        
-				 
+
+        <!-- https://mvnrepository.com/artifact/org.rdfhdt/hdt-java-package -->
+
+        <dependency>
+            <groupId>org.rdfhdt</groupId>
+            <artifactId>hdt-java-package</artifactId>
+            <type>pom</type>
+            <exclusions>
+                <exclusion>
+                    <groupId>junit</groupId> <!-- Exclude Project-D from Project-B -->
```

Comment:
not is this case, because this concerns only this dependency. If this is on parent, will breake other modules

Reply:
No, that is not correct since it is only excluding the junit dependency defined within hdt but not the JUnit dependency definition that we have in our pom files. Apart from that, the exclusion is already part of the parent pom, if I see it correctly. So you can simply remove the exclusion from this pom file.